fix: correct file permissions for the apt cache (#1194)
canonical/craft-parts: docs/reference/changelog.rst

- `#1007 <https://github.com/canonical/craft-parts/issues/1007>`_ When wild cards
  were used in an ``organize`` source path, an error would occur if files mapped to
  themselves. These cases are now ignored.

> - A warning is no longer emitted by apt when downloading to the stage package cache.

Reviewer: ```suggestion
- Suppress warnings from APT when downloading to the stage package cache.
```

The warning is still emitted, just not captured, right?

Reviewer 2: I see the word "suppress" as meaning that it still does something wrong that warrants a warning, we just hide it. This code change makes no warning-triggering behavior happen anymore. So I don't think it would be a good word choice
